Makefile.dapper

```diff
@@ -11,7 +11,7 @@
 	@mv .dapper.tmp .dapper
 
 %: .dapper
-	./.dapper -m bind $(MAKE) $@ $(MAKEFLAGS)
+	./.dapper -m bind make $@ $(MAKEFLAGS)
```

Comment:
```suggestion
	+./.dapper -m bind make $@ $(MAKEFLAGS)
```

The `+` is necessary if we don’t use the `MAKE` variable, so that `make -n` and the like will continue working. See [How the `MAKE` Variable Works](https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html) for details.

Reply:
What's that black + magic? :)